### PR TITLE
Add tests for VideoComposer.

### PR DESCRIPTION
**Pull request by BartWojtowicz**

### CODE REVIEW

File: tests/test_compose.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from videopython.base.compose import VideoComposer
+from videopython.base.transforms import TransformationPipeline
+from videopython.base.transitions import FadeTransition, InstantTransition
+
+
+def test_vanilla_compose(small_video):
+    videos_to_compose = [small_video, small_video]
```
Comment (bartoszrud) [inline tests/test_compose.py:9]: I would change this to two different videos so we can detect potential issues when only one video is used. In the current setting, the last frame will still be the same in such cases.

File: tests/test_compose.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    transition = InstantTransition()
+    composer = VideoComposer(transformation_pipeline, transition)
+    composed_video = composer.compose(videos_to_compose)
+    assert composed_video.total_seconds == 2 * small_video.total_seconds
+    assert composed_video.fps == small_video.fps
+    assert np.all(composed_video.frames[0] == small_video.frames[0])
+    assert np.all(composed_video.frames[-1] == small_video.frames[-1])
+
+def test_fade_compose_100_frames(small_video):
+    videos_to_compose = [small_video, small_video]
```
Comment (bartoszrud) [inline tests/test_compose.py:20]: Same as in the test above.

**Review by bartoszrud — CHANGES_REQUESTED** [review]

File: src/videopython/base/compose.py [hunk 3]
```diff
@@ -11,20 +14,43 @@ class VideoComposer:
 
     def __init__(
         self,
-        transformation_pipeline: TransformationPipeline,
-        transition: Transition | list[Transition],
+        transformation_pipeline: TransformationPipeline | None = None,
+        transition: Transition = InstantTransition,
     ):
```
[the remark this answers is not in the document]
Comment (BartWojtowicz) [reply src/videopython/base/compose.py]: Changed to instantiated object

**Review by bartoszrud — APPROVED** [review]
Great work!

File: tests/test_compose.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+from videopython.base.transitions import FadeTransition, InstantTransition
+
+
+def test_vanilla_compose(small_video):
+    videos_to_compose = small_video.split()
+    transformation_pipeline = TransformationPipeline([])
+    transition = InstantTransition()
+    composer = VideoComposer(transformation_pipeline, transition)
+    composed_video = composer.compose(videos_to_compose)
+    assert composed_video.total_seconds == 2 * small_video.total_seconds
```
Comment (bartoszrud) [inline tests/test_compose.py:14]: After splitting videos, test no longer valid

Comment (BartWojtowicz) [reply tests/test_compose.py]: Ahh, the small_video no longer existed after split, fixed now 😌

**Pull request merged**